From 02efbed152a3a9e364cdb73628af0aed0f5d6f7a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Caol=C3=A1n=20McNamara?= <caolanm@redhat.com>
Date: Wed, 4 Dec 2013 16:48:08 +0000
Subject: [PATCH] Resolves: rhbz#1038189 refresh printer list when print dialog
 launched

which is the the same thing the printer settings dialog does

Change-Id: I36fb9f057b66faa8c9fce5a9934ed2fb70786363
---
 vcl/source/window/printdlg.cxx | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/vcl/source/window/printdlg.cxx b/vcl/source/window/printdlg.cxx
index b8cb188..21941ca 100644
--- a/vcl/source/window/printdlg.cxx
+++ b/vcl/source/window/printdlg.cxx
@@ -602,6 +602,8 @@ PrintDialog::PrintDialog( Window* i_pParent, const boost::shared_ptr<PrinterCont
     // init reverse print
     maJobPage.mpReverseOrderBox->Check( maPController->getReversePrint() );
 
+    Printer::updatePrinters();
+
     // fill printer listbox
     const std::vector< OUString >& rQueues( Printer::GetPrinterQueues() );
     for( std::vector< OUString >::const_iterator it = rQueues.begin();
-- 
1.8.3.1

